aws/ssm/nanny.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+
+curl -O https://raw.githubusercontent.com/harmony-one/harmony/master/scripts/wallet.sh
```

Comment:
why download wallet in nanny script?

Reply:
again, this `nanny` script will be discarded in the future.